Sources/eppo/EppoClient.swift

```diff
@@ -52,52 +52,68 @@ public class EppoClient {
         let (urlData, _) = try await httpClient.get(url);
         self.flagConfigs = try JSONDecoder().decode(FlagConfigJSON.self, from: urlData);
     }
-    
-    public func getAssignment(
-        _ subjectKey: String,
-        _ flagKey: String,
-        _ subjectAttributes: SubjectAttributes) throws -> String?
-    {
-        return try getInternalAssignment(subjectKey, flagKey, subjectAttributes, false)?.stringValue()
-    }
 
     public func getBoolAssignment(
-        _ subjectKey: String,
-        _ flagKey: String,
-        _ subjectAttributes: SubjectAttributes = SubjectAttributes()) throws -> Bool?
+        flagKey: String,
+        subjectKey: String,
```

Comment:
how important is it to flip the order of subjectKey and flagKey here? Adding the required labels makes this less likely, but I worry that this might be easy to miss in a refactoring